dotCMS/src/main/java/com/dotmarketing/startup/StartupAPIImpl.java

```diff
@@ -0,0 +1,72 @@
+package com.dotmarketing.startup;
+
+import com.dotcms.business.WrapInTransaction;
+import com.dotcms.util.CollectionsUtils;
+import com.dotcms.util.ReflectionUtils;
+import com.dotmarketing.exception.DoesNotExistException;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.TaskLocatorUtil;
+import com.google.common.collect.ImmutableList;
+
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * Default implementation.
+ * @author jsanca
+ */
+public class StartupAPIImpl implements StartupAPI {
```

Comment:
I would name it `StartupTaskAPIImpl ` is very specific to startup tasks and just `Startup` could create confusion about the purpose of the class.